frontend/templates/process.html

```diff
@@ -25,26 +25,38 @@
     <div class="process-form-block">
 
         <form id="process-form" name="process-form" data-name="process-form" class="process-form">
+                <div class="description-columns">
+                    <div class="div-wrapper">
+                        <div class="left-div">
+                            
+                            <label for="process-name-textarea"
+                                    class="entry-label">Name des Prozesses
+                            </label>
+                            <input  type="text"
+                                    name="process-name-textarea"
+                                    maxlength="300"
+                                    id="process-name-textarea"
+                                    class="textfield">
 
-            <div class="description-columns">
-                <div class="div-wrapper">
-                    <div class="left-div">
-                        <label for="process-name-textarea"
-                               class="entry-label">Name des Prozesses
-                        </label>
-                        <input type="text"
-                               name="process-name-textarea"
-                               maxlength="300"
-                               id="process-name-textarea"
-                               class="textfield">
-                        <label for="process-beschreibung-textarea"
-                               class="entry-label">Beschreibung des Prozesses
-                        </label>
+                            <label for="process-responsible-person-textarea"
+                                    class="entry-label">Responsible Person
```

Comment:
Warum macht ihr es hier nicht direkt richtig? Klar, es gibt keinen Issue dazu, aber euer Feld gibt es da ja noch garnicht. Also bitte entsprechend anpassen.

Reply:
wenn dann würden wir ja schon das gesamte issue lösen und wir dachten dass da dann eine person den kleinen task übernehmen kann

Reply:
also wir können auch das issue in diesem branch lösen...

Reply:
Bitte einfach nur das eine Feld korrigieren, man muss ja nicht extra nicht falschen Code schreiben. Und ggf. hat Erik ja schon angefangen. Danke :)

Reply:
wir haben es nicht absichtlich falsch gemacht sondern das ändern der json, sowie das erstellen einer jac´vascipt funktion einfach aus unserem scope excluded haben weil das im bugfix sowieso erstellt werden muss
aber wie gesagt wir können natürlich die arbeit doppelt machen, bzw das einfach mergen und dann macht erik einheitlich alle drei bezeichner